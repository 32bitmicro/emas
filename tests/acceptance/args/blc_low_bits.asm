blc 0xff01
