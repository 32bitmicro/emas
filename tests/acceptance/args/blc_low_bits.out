acceptance/args/blc_low_bits.asm:1:11: BLC argument may only have left byte bits set
